js/store.js

```diff
@@ -11,6 +10,91 @@ const store = {
       }
     }
     browser.runtime.onMessage.addListener((m) => store.messageHandler(m));
+
+    // get Disconnect Entity List from shavar-prod-lists submodule
+    let whiteList;
```

Comment:
Ah I thought of one approach that worked:
```javascript
await this.getWhiteList();
```
And have `this.getWhiteList` assign to `this.firstPartyWhiteList` and `this.thirdPartyWhiteList` instead of returning values to the `this.init` method.